pre-commit/README.md

```diff
@@ -1,7 +1,7 @@
 # pre-commit
 
-This action checks if the PR passes [pre-commit](https://pre-commit.com/).  
-For public repositories, using [pre-commit.ci](https://pre-commit.ci/) is recommended.  
+This action checks if the PR passes [pre-commit](https://pre-commit.com/).
+For public repositories, using [pre-commit.ci](https://pre-commit.ci/) is recommended.
```

Comment:
Could you please revert unnecessary changes?

Reply:
Done https://github.com/autowarefoundation/autoware-github-actions/pull/237/commits/c5bb04ac9c23c2bd57aefa3e78d8692e1659d969.